pkg/alpine/mirrors/3.18/main

```diff
@@ -1 +0,0 @@
-mkinitfs
```

Comment:
Shouldn't we remove the entire 3.18 dir?

Reply:
git doesn't consider empty directories. This commit was reverted using git revert. So I think we are safe here.